Package.swift

```diff
@@ -4,10 +4,13 @@ import PackageDescription
 
 let package = Package(
     name: "S5TF",
+    platforms: [
+        .macOS(.v10_13)
```

Comment:
Could we add Linux as a platform as well?

Reply:
This is a minimum macOS requirement. It should work on Linux as well because the tests pass: https://github.com/rickwierenga/S5TF/runs/407740256.

See https://github.com/tensorflow/swift-models/blob/master/Package.swift#L8.

Reply:
Sounds good :+1: